wolfssl/wolfcrypt/settings.h

```diff
@@ -2094,9 +2094,6 @@ extern void uITRON4_free(void *p) ;
     #if defined(HAVE_IO_POOL) || defined(XMALLOC_USER) || defined(NO_WOLFSSL_MEMORY)
          #error static memory cannot be used with HAVE_IO_POOL, XMALLOC_USER or NO_WOLFSSL_MEMORY
     #endif
-    #if !defined(USE_FAST_MATH) && !defined(NO_BIG_INT)
```

Comment:
This check needs to be here for the integer.c case. The logic is a bit long, but should look like this:

```
/* FastMath or SP with no malloc are allowed with static memory */
#if (!((defined(WOLFSSL_HAVE_SP_RSA) || defined(WOLFSSL_HAVE_SP_DH) || \
        defined(WOLFSSL_HAVE_SP_ECC)) && defined(WOLFSSL_SP_NO_MALLOC) || \
    (!defined(USE_FAST_MATH) & !defined(NO_BIG_INT)))
```